docs/content/concepts/dagster-pipes/aws-glue.mdx

```diff
@@ -0,0 +1,146 @@
+
+---
+
+## Prerequisites
+
+- **In the orchestration environment**, you'll need to:
+
+  - Install the following packages:
+
+    ```shell
+    pip install dagster dagster-webserver dagster-aws
+    ```
+
+    Refer to the [Dagster installation guide](/getting-started/install) for more info.
+
+  - **An existing boto3 client that can authenticate to AWS.** If you don't have this set up already, refer to the [boto3 quickstart](https://boto3.amazonaws.com/v1/documentation/api/latest/guide/quickstart.html).
+
+- **In AWS**:
+
+  - An existing AWS account
+  - An AWS Glue job with a Python 3.8+ runtime environment
+
+---
+
+## Step 1: Provide the dagster-pipes module
+
+Provide the `dagster-pipes` module to the AWS Glue job either by installing it in the Glue job environment or packaging it along with the job script.
+
+---
+
+## Step 2: Add dagster-pipes to the Glue job
+
+Call `open_dagster_pipes` in the Glue job script to create a context that can be used to send messages to Dagster:
+
+```python file=/guides/dagster/dagster_pipes/glue/glue_script.py
+import boto3
+from dagster_pipes import (
+    PipesCliArgsParamsLoader,
+    PipesS3ContextLoader,
+    PipesS3MessageWriter,
+    open_dagster_pipes,
+)
+
+client = boto3.client("s3")
+context_loader = PipesS3ContextLoader(client)
+message_writer = PipesS3MessageWriter(client)
+params_loader = PipesCliArgsParamsLoader()
+
+
+def main():
+    with open_dagster_pipes(
+        context_loader=context_loader,
+        message_writer=message_writer,
+        params_loader=params_loader,
+    ) as pipes:
+        pipes.log.info("Hello from AWS Glue job!")
+
+
+if __name__ == "__main__":
+    main()
+```
+
+## Step 3: Add the PipesGlueClient to Dagster code
+
+In the Dagster asset/op code, use the `PipesGlueClient` resource to launch the job:
+
+```python file=/guides/dagster/dagster_pipes/glue/dagster_code.py startafter=start_asset_marker endbefore=end_asset_marker
+import os
+
+# dagster_glue_pipes.py
+import boto3
+from dagster_aws.pipes import PipesGlueClient
+
+from dagster import AssetExecutionContext, asset
+
+
+@asset
+def glue_pipes_asset(
+    context: AssetExecutionContext, pipes_glue_client: PipesGlueClient
+):
+    return pipes_glue_client.run(
+        context=context,
+        job_name="Example Job",
+        arguments={"some_parameter_value": "1"},
+    ).get_materialize_result()
+```
+
+This will launch the AWS Glue job and monitor its status until it either fails or succeeds. A job failure will also cause the Dagster run to fail with an exception.
+
+---
+
+## Step 4: Creating Dagster definitions
```

Comment:
```suggestion
## Step 4: Create Dagster definitions
```

Sorry if I did this, but lets do this instead so it's the same as the other headings